CIP-XXXX/README.md

```diff
@@ -0,0 +1,60 @@
+---
+CIP: ?
+Title: CIP-30 ext: Bulk transaction signing
+Category: Wallets
+Status: Proposed
+Authors:
+    - Martín Schere
+    - Ola Ahlman <ola.ahlman@tastenkunst.io>
+Implementors: 
+    - JPG Store
+    - Eternl
+    - Typhon
+Discussions:
+    - https://github.com/cardano-foundation/CIPs/pull/443
+Created: 2023-09-03
+License: CC-BY-4.0
+---
+
+## Abstract
+This CIP extends [CIP-30 (Cardano dApp-Wallet Web Bridge)](https://cips.cardano.org/cips/cip30/) to provide an additional endpoint for dApp to sign multiple transactions in bulk.
+
+## Motivation: why is this CIP necessary?
+Currently, there is no way to sign multiple transactions in bulk, and the experience of signing a chain of transactions is suboptimal. We propose the addition of a signTxs endpoint that enable wallets to create an array of interconnected transactions and sign them all at once.
+
+## Specification
```

Comment:
Would it also make sense to introduce a bulk transaction submission? something like `.submitTxs()`?

Reply:
That would indeed make sense to make this CIP extension more complete. Submitting multiple transactions should work a bit differently compared to signTxs when it comes to error handling I think. While I mentioned that tx signing should sign all or none, for submitting, I would argue that it should try to submit all, and respond with an array mapping to input transaction array with either `null or string[]` as value, with null representing successful submit and string array the error response. This for each entry in array.
`Array<null|string[]>()`

Does this make sense?

Reply:
A `submitTxs` endpoint has now been added to the specification. I implemented it slightly differently than above message so please review actual README and not my comment.